component/common/src/main/java/org/wso2/carbon/extension/identity/emailotp/common/constant/Constants.java

```diff
@@ -0,0 +1,159 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
update the license header

Reply:
fixed in 99ef19caaf9cd3d8a29b38a5d0c58726f2c97e49